server/json-logger.cpp

```diff
@@ -224,6 +225,16 @@ void JsonLogger::write_log(vk::string_view message, int type, int64_t created_at
   if (tags_available_) {
     json_out_it->append_raw(tags_);
   }
+  if (process_type == ProcessType::master) {
+    json_out_it->append_key("process_type").append_string("master");
+  } else if (process_type == ProcessType::general_worker) {
+    json_out_it->append_key("process_type").append_string("general_worker");
+    json_out_it->append_key("worker_unique_index").append_integer(logname_id);
+  } else if (process_type == ProcessType::job_worker) {
+    json_out_it->append_key("process_type").append_string("job_worker");
+    json_out_it->append_key("worker_unique_index").append_integer(logname_id);
```

Comment:
let's call it `logname_id`
```suggestion
    json_out_it->append_key("logname_id").append_integer(logname_id);
  } else if (process_type == ProcessType::job_worker) {
    json_out_it->append_key("process_type").append_string("job_worker");
    json_out_it->append_key("logname_id").append_integer(logname_id);
```